specification/ai/DocumentIntelligence/models.tsp

```diff
@@ -1005,7 +1005,7 @@ model DocumentLanguage {
     Detected language.  Value may an ISO 639-1 language code (ex. \"en\", \"fr\")
     or BCP 47 language tag (ex. \"zh-Hans\").
     """)
-  locale: string;
```

Comment:
the current tool can detect indirect changes in type, which can't be detected from old tool.
the current output is confusing, since it use the same format to old tool.
since sdk suppression file is not intended to live long, and `breakingChangeItems` decouple the changelog we write to `CHANGELOG.MD` and sdk suppresion files - `breakingChangeItems` should be match sdk suppression file, we can consider improve all changelog item by adding more change details in next pr, like "xxx is changed from A to B", "Function Func has new signature '...' due to A changed to B causing ParaA to ParaB" - the old tool don't tell you what-changes-to-what many cases